dffml/source/ini.py

```diff
@@ -0,0 +1,63 @@
+from configparser import ConfigParser
+
+from ..record import Record
+from .file import FileSource
+from ..base import config, field
+from .memory import MemorySource
+from ..util.entrypoint import entrypoint
+
+
+@config
+class INISourceConfig:
+    filename: str
+    feature: str = field("Name of the feature the data will be referenced as")
```

Comment:
I don't think to meant to keep this line (the GitHub actions logs are yelling about it)

```suggestion
```